docs/contents.md

```diff
@@ -47,8 +47,8 @@ When creating notification rules, selecting **Manual Input** as a notification t
 
 | Playbook Name          | Description                                                                                                                                   |
 |:-----------------------|:----------------------------------------------------------------------------------------------------------------------------------------------|
-| Create Indicator       | Extracts indicators from the text provided as input                                                                                           |
-| Create Alert           | Creates an alert based on the values provided in the form                                                                                     |
+| Create Indicator       | Extracts indicators from the text provided as input. Gets triggered on createIndicator command when used in Microsoft Teams                   |
```

Comment:
Extracts indicators from the text sent on the chat interface and is triggered when `createIndicator` command is sent through Microsoft Teams.